tests/_baseTest.js

```diff
@@ -5,7 +5,7 @@ var _ = require('lodash');
 function _BaseKeabbyTest(baseConfig) {
 
   var krabbyTest = function(config) {
-    this.config = _.extend(baseConfig, config);
+    this.config = _.extend({}, baseConfig, config);
```

Comment:
Almost all of the tests fail without this. I know this is already in #20 so we can either merge that first and remove from this branch or close it out in favor of this one.